### ISSUE TITLE
Data present in the table getting truncated after zooming page to 200% : A11y_.NetAspireDashboard_Resources_Resources_Zoom

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspire#3176]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #Win11; #ChromiumEdge; #FTP; #DesktopWeb; #.NET Core; #SH_.Net Aspire Dashboard_Web_Mar2024; #WCAG1.4.4; #Zoom;

### Prerequisites:
Set display resolution to 1280*768.

### Environment Details:
Application Name: .Net Aspire Dashboard
URL: https://localhost:19888/
Chrome Version 122.0.2365.80 (Official build) (64-bit)
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Hit the URL: http://localhost:15888/
2. TAB to "Resources" tab item and hit select the tab by pressing Enter key.
3. Zoom the page to 200% and observe the issue.

### Actual Results:
Data present in the table getting truncated after zooming page to 200%.

**Similar issue is also observed in throughout the application for the table screen.**

### Expected Results:
All data and controls on the page should be clearly visible when page is zoomed to 200%.

### User Impact:
Low vision users will not be able to see the content present in the page clearly and will miss the information. 

### Attachment:

![3176_A11y_ NetAspireDashboard_ResourceTab_AfterZoom](https://github.com/dotnet/aspire/assets/90901519/8d32b705-5a71-4356-8200-fe8f1b5632b2)
![3176_A11y_ NetAspireDashboard_ResourceTab_BeforeZoom](https://github.com/dotnet/aspire/assets/90901519/c49ed100-4336-4a41-978f-7175c815ec1b)
![3176_A11y_ NetAspireDashboard_StructuredLogs_Table_BeforeResizing](https://github.com/dotnet/aspire/assets/90901519/ab448fd1-c8d3-476c-af61-2c3f6d881813)
![3176_A11y_ NetAspireDashboard_StructuredLogs_Table_AfterResizing](https://github.com/dotnet/aspire/assets/90901519/97cb6991-7099-4513-bb7e-0cc2703867a7)

**Comment by adamint** [comment]
@PoojaNamde can you give guidance on how to display this content? Some rows, such as the "basketcache-commander" resource, won't have the width to display all row content without decreasing the font size

**Comment by PoojaNamde** [comment]
@adamint Here we can apply the functionality of resizing column width, by adjusting the column width we can see the data clearly.

**Comment by adamint** [comment]
@PoojaNamde did you mean to send a screenshot? We are constrained by minimum widths of columns as well

**Comment by PoojaNamde** [comment]
@adamint Then tooltips should be provided for all the data present in the table and all the tooltips should be accessible using keyboard.

**Comment by adamint** [comment]
@PoojaNamde can you review this

**Comment by adamint** [comment]
@PoojaNamde could you re-test this issue, please? Thank you.

**Comment by kvenkatrajan** [comment]
Work is in progress. On track for August commitment.
